src/views/v2/rsvp/form/form.php

```diff
@@ -31,6 +31,8 @@
 
 		<div class="tribe-tickets__rsvp-form-content tribe-tickets__form">
 
+			<?php $this->template( 'v2/rsvp/form/error', [ 'rsvp' => $rsvp, 'going' => $going ] ); ?>
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 13d9224df7eed81565dc58d015343bfb</sub>